pyproject.toml

```diff
@@ -72,6 +72,12 @@ extend-select = [
 [tool.ruff.lint.flake8-tidy-imports]
 ban-relative-imports = "all"
 
+[tool.ruff.lint.flake8-type-checking]
+exempt-modules = [
```

Comment:
why?

Reply:
The things in `collections.abc` are for me the same things as in  `typing`: We only ever import them for type hints. So because we don't put `typing` in a `if TYPE_CHECKING` block, we shouldn't do this for `collections.abc` as well.

Reply:
I would go with the default, even if that moves the import into the typechecking block. Doesn't sound like a bad thing to me.

Reply:
I find it ugly, but it isn't worth to debate to long.